dependencies.repos

```diff
@@ -6,3 +6,11 @@
     local-name: opw_kinematics
     uri: https://github.com/Jmeyer1292/opw_kinematics.git
     version: 0.5.0
+- git:
+    local-name: fcl
+    uri: https://github.com/flexible-collision-library/fcl.git
+    version: 0.6.1
```

Comment:
Why did you have to add fcl?

Reply:
Version 0.6.1 is not distributed and `tesseract_collision` requires at least that version

Reply:
Can we not install it from ros?

Reply:
My hope is to make this a fully ROS-independent build to support either ROS1 or ROS2 downstream repos, so no...

Reply:
Ah, makes sense.